tools/fuchsia/build_fuchsia_artifacts.py

```diff
@@ -224,7 +224,7 @@ def GetRunnerTarget(runner_type, product, aot):
 
 def GetTargetsToBuild(product=False, additional_targets=[]):
   targets_to_build = [
-      'flutter/shell/platform/fuchsia:fuchsia',
+      'flutter', 'fuchsia_tests'
```

Comment:
so the fuchsia_tests target was deliberately left out because building the fars can take a really long time, so a decision was made to explicitly require the tests to be built.

Can we add a flag to `build_fuchsia_artifacts.py` instead, something like `--unit-tests` which will add this target instead of making it the default?

Reply:
Sure thing

Reply:
Actually, additional_targets can handle this so I'll just use that instead